Kimjin/3주차 과제/과제1/style.css

```
@@ -0,0 +1,83 @@
+body {
+  background-color: rgb(139, 186, 204);
+}
+
+div {
+  margin: 50px auto;
+  width: 500px;
+  display: flex;
+  justify-content: center;
+  border: 3px solid white;
+  border-radius: 10px;
+  background-color: rgb(216, 243, 255);
+}
+
+.box2 {
+  height: 150px;
+  margin-top: -30px;
+}
+
+.one1 {
+  height: 45px;
+  width: 45px;
+  border-radius: 50%;
+  border: none;
+  background-color: rgb(65, 65, 150);
+  transform: all 1s ease-in;
+}
+
+.one2 {
+  height: 45px;
+  width: 45px;
+  border-radius: 50%;
+  border: none;
+  background-color: rgb(65, 65, 150);
+  transform: all 1s ease-in;
+}
+
+.one3 {
+  height: 45px;
+  width: 45px;
+  border-radius: 50%;
+  border: none;
+  background-color: rgb(65, 65, 150);
+  transform: all 1s ease-in;
+}
+
+.one4 {
+  height: 45px;
+  width: 45px;
+  border-radius: 50%;
+  border: none;
+  background-color: rgb(65, 65, 150);
+  transform: all 1s ease-in;
+}
+
+.one5 {
+  height: 45px;
+  width: 45px;
+  border-radius: 50%;
+  border: none;
+  background-color: rgb(65, 65, 150);
+  transform: all 1s ease-in-out;
+}
+
+.one1:active {
+  background-color: rgb(216, 243, 255);
+}
+
+.one2:active {
+  background-color: rgb(216, 243, 255);
+}
+
+.one3:active {
+  background-color: rgb(216, 243, 255);
+}
+
+.one4:active {
+  background-color: rgb(214, 97, 97);
+}
+
+.one5:active {
+  background-color: rgb(216, 243, 255);
+}
```

중복되는 코드들을 클래스를 이용해서 합치면 이렇게 줄일 수 있어요!
```suggestion
.circle {
  height: 45px;
  width: 45px;
  border-radius: 50%;
  border: none;
  background-color: rgb(65, 65, 150);
  transition: all 1s ease-in;
}

.circle:active {
  background-color: rgb(216, 243, 255);
}

.point-circle:active {
  background-color: rgb(214, 97, 97);
}
```